torch/lib/c10d/ProcessGroupNCCL.cpp

```diff
@@ -287,90 +287,107 @@ std::vector<std::shared_ptr<NCCLComm>>& ProcessGroupNCCL::getNCCLComm(
   return devNCCLCommMap_[devicesKey];
 }
 
-// Helper function that checks the input and output tensors for validity
-void ProcessGroupNCCL::tensorCheckHelper(
-    const std::vector<at::Tensor>& input,
-    const std::vector<at::Tensor>& output,
-    int outputOverInput) {
-  if (input.size() * outputOverInput != output.size()) {
-    throw std::runtime_error(
-        "Input tensor sequence should have the same "
-        "number of tensors as the output tensor sequence");
-  }
+namespace {
 
-  if (input.size() == 0) {
-    throw std::runtime_error("The number of input tensors should not be zero");
+/*
+ * Check that all `tensors' have the same type and shape and are distributed
+ * across distinct GPUs.
+ */
```

Comment:
All comments in this directory use `//` style.

Did a quick check in the rest of the repository and the block comment is not prevalent elsewhere either:

```
$ grep -ir '^//' * | wc -l
3179
$ grep -ir '^/\*' * | wc -l
33
```